coala_quickstart/Strings.py

```diff
@@ -33,3 +33,13 @@
              "all its contents but exclude your .git directory and all "
              ".o files. To do this, simply give `src/` for the first "
              "question and `.git/**,**/*.o` for the second question.")
+
+# Start ignoring LineLengthBear
+bear_docs_url = "https://github.com/coala-analyzer/bear-docs/blob/master/README.rst#supported-languages"
+# Stop ignoring
+bear_help = ("A coala bear is a plugin that contains the actual algorithm. "
+             "It have its own native code or use an external linter. It "
```

Comment:
It **has**

\+ native? I wouldn't talk about native code and python together :3